progression/src/main/java/me/mykindos/betterpvp/progression/progression/perks/DropMultiplier.java

```diff
@@ -0,0 +1,20 @@
+package me.mykindos.betterpvp.progression.progression.perks;
+
+import me.mykindos.betterpvp.core.config.Config;
+import me.mykindos.betterpvp.core.utilities.UtilMath;
+import net.kyori.adventure.text.Component;
+import org.bukkit.Bukkit;
+
+import java.util.Random;
+
+public interface DropMultiplier {
+
+    default int getExtraDrops(double chance) {
+        int multiplier = (int) Math.floor(chance/100);
```

Comment:
Unless the chance is over 100 (which it isn't for the first 400 levels after `minLevel` (100 / 0.25), `chance/100` will return a decimal value under 1 that will get floor to zero and give them a 0 multiplier for the first 400 levels.

Reply:
Intended. Random chance is handled below, this is to allow for chance over 100.